docs/windows.md

```diff
@@ -0,0 +1,64 @@
+# Working on Windows
+
+jj works the same on all platforms, but there are some caveats that Windows
```

Comment:
Nit: Use `jj` or Jujutsu throughout the whole doc. Atm it is not consistent with the rest.

Reply:
Updated to "Jujutsu" everywhere.